src/main/java/org/symphonyoss/symphony/messageml/elements/Form.java

```diff
@@ -45,4 +50,18 @@ protected void buildAttribute(org.w3c.dom.Node item) throws InvalidInputExceptio
   public org.commonmark.node.Node asMarkdown() {
     return new FormNode();
   }
-}
\ No newline at end of file
+
+  /**
+   * Assert that children with any of the informed types do not exceed the maximum count allowed.
+   *
+   * @param elementTypes
+   * @throws InvalidInputException
+   */
+  private void assertChildrenNotExceedingMaxCount(Collection<Class<? extends Element>> elementTypes) throws InvalidInputException {
```

Comment:
Put this into the Element class and parameterize the maxCount into the method as well so that other elements can use the same logic later on if needed.

Reply:
Also, explain briefly the parameters on the javadoc